src/AndorSIF.jl

```diff
@@ -184,20 +190,27 @@ function FileIO.load(fs::Stream{format"AndorSIF"})
     for i = 1:frames
         _ = readline(io)
     end
-    offset = position(io) # start of the actual pixel data, 32-bit float, little-endian
 
-    pixels = read(io, Float32, width, height, frames)
+    # two additional lines?
```

Comment:
This is the only part that seems concerning, assuming that it nominally worked before. (I've never tested this myself.) Since it's "weird," is there way to do a sanity check and make sure you're not accidentally consuming `offset` (below)?

Reply:
Including either one or both lines yields non-integer values for the test files I have at hand. Also, the data looks correct and if I remember correctly, I checked it against tif files generated alongside the sif files.